### DISCUSSION TITLE
Toggleable Early/Late Colors on all notes below Great instead of looking on the score meter.

### DISCUSSION THREAD

**Discussion by chara22322** [ppy/osu#22622] [Ideas] [answered]
It is easier to judge if you are either too fast or slow on notes when you don't need to deviate your attention from the map mid-play, or rewatching the replay.
My suggestion for colors is: Orange for too fast, Cyan/Light Blue for too slow.

**Comment by mk56-spn** [comment, marked as answer]
https://github.com/ppy/osu/discussions/22603 this was asked yesterday, and not for the first time.
